apps/design/backend/src/env.d.ts

```diff
@@ -1,13 +1,15 @@
 declare namespace NodeJS {
   /** process.env typings */
   export interface ProcessEnv {
+    readonly AUTH_ENABLED?: string;
     readonly AUTH0_CLIENT_ID?: string;
     readonly AUTH0_ISSUER_BASE_URL?: string;
     readonly AUTH0_SECRET?: string;
     readonly BASE_URL?: string;
     readonly CI?: string;
     readonly DEPLOY_ENV: 'development' | 'staging' | 'production';
     readonly NODE_ENV: 'development' | 'production' | 'test';
+    readonly ORG_ID_VOTINGWORKS?: string;
```

Comment:
For feature flagging, we're gonna need both the VotingWorks and NH ideas available in the frontend. Or I guess we could assume any org that's not VotingWorks is NH. I was imagining we just hardcode the org IDs into the codebase for now. What do you think makes the most sense? Env vars for both?

Reply:
Yeah, I was thinking we can just assume all orgs are NH for now, unless explicitly special-cased (VotingWorks, maybe SLI).
Only using an env var here because the org ID will be different across dev/staging/prod Auth0, but we could go with hardcoded prog IDs for the NH orgs, if that's easier - might just be tough keeping them in sync with what we create in Auth0 over the next few days

Reply:
Ok, I'll try building on top of what you have here. I think it should work, but we can see how it goes.